lib/ddtrace.rb

```diff
@@ -61,3 +61,15 @@ class Railtie < Rails::Railtie
         'auto-instrumentation for core components will be disabled.'
   end
 end
+
+if defined?(Sinatra::VERSION)
+  vs = Gem::Version.new(Sinatra::VERSION)
+  min_vs = Gem::Version.new('1.4.0')
+  if vs >= min_vs
+    require 'ddtrace/contrib/sinatra/tracer'
```

Comment:
I think it's better to prevent any kind of side effect, so this import should not add anything to users' app. We may follow the Rails approach so that to use our integration users have to:
* add the gem
* explicitly call a method or use a kind of ``auto_instrument`` config setting